conformance/utils/suite/suite.go

```diff
@@ -35,16 +36,21 @@ type ConformanceTestSuite struct {
 	Debug            bool
 	Cleanup          bool
 	BaseManifests    string
+	Applier          kubernetes.Applier
 }
 
 // Options can be used to initialize a ConformanceTestSuite.
 type Options struct {
-	Client           client.Client
-	GatewayClassName string
-	Debug            bool
-	Cleanup          bool
-	RoundTripper     roundtripper.RoundTripper
-	BaseManifests    string
+	Client               client.Client
+	GatewayClassName     string
+	Debug                bool
+	Cleanup              bool
+	RoundTripper         roundtripper.RoundTripper
+	BaseManifests        string
+	NamespaceAnnotations map[string]string
+	// ValidListenerPorts holds a list of ports assignable to Gateway listeners.
+	// If empty, every listener port will be left as is.
+	ValidListenerPorts []v1alpha2.PortNumber
```

Comment:
Was there any intention to allow these to be configured with flags?

Reply:
We're using the conformance tests as a library so I didn't bother.